src/entitytypes.js

```diff
@@ -976,6 +976,35 @@ SlugIdArray.prototype.indexOf = function(slug) {
   return -1;
 };
 
+/** Determines whether it includes a certain element, returning true or false as appropriate. */
+SlugIdArray.prototype.includes = function(slug) {
+  return this.buffer.includes(slugIdToBuffer(slug));
```

Comment:
Won't work... you need to ensure alignment...

Reply:
@jonasfj Could you provide me a test case that won't work. I'm not sure I understand what you mean by alignment. Maybe I could just do:

```js
SlugIdArray.prototype.includes = function(slug) {
  return this.indexOf(slug) !== -1 ? true : false;
};
```

Reply:
See what `indexOf` does... 

You have an array of bytes where every 16 bytes is a slugid... you can't ask if the array contains a slugid, because if it does it still has to be align...

Imagine if slugid was just two bytes and I had array: `[a,b,c,d]` it contains ids: `ab` and `cd`, not `bc` which you would get if you just asked if it includes without considering alignment...

Reply:
Ah, I see. Basically you want to make sure the match isn't one with bytes overlapping with the neighbouring slug. `indexOf` does this check with `index % SLUGID_SIZE === 0`. I could simply use the logic I suggested above then.